File: Java/Framework/src/metrics/usability/TrainingTime.java
package metrics.usability;

public class TrainingTime {
    private long start;
    private long finish;
    public void startTraining(){
        start = System.currentTimeMillis();
    }
    public void endTraining(){
        finish = System.currentTimeMillis();
    }
    public long getTrainingTime(){
        return finish - start;
    }
}

File: Dart/Framework/metrics/usability/training_time.dart
class TrainingTime {
  late int start;
  late int finish;

  void startTraining() {
    start = DateTime.now().millisecondsSinceEpoch;
  }

  void endTraining() {
    finish = DateTime.now().millisecondsSinceEpoch;
  }

  int getTrainingTime() {
    return finish - start;
  }
}
